Automattic/newspack-blocks
# Add a first pass at German translations to the blocks

### All Submissions:


### Changes proposed in this Pull Request:

This PR adds a first pass of German translations to the block.

Closes #350 .

### How to test the changes in this Pull Request:

1. Apply the PR.
2. Navigate to Settings > General and switch the language to German.
3. Edit a page, and add a homepage block that displays the author, and a donate block.
4. In the editor confirm both are showing translated strings (`von` and `und` for the homepage block, and multiple -- but not all -- strings in the donate block:

![image](https://user-images.githubusercontent.com/177561/73410065-03981100-42b6-11ea-83d9-a8b31314c0c7.png)

![image](https://user-images.githubusercontent.com/177561/73410073-0b57b580-42b6-11ea-93a4-f55950b32d54.png)

5. Save and publish the page.
6. Confirm the same translations appear on the published page.

### Other information:
